### PR TITLE
Update new sidecar versions and 1.19 changelog

### PR DESCRIPTION
**Pull request by msau42**

### CODE REVIEW

File: book/src/external-attacher.md [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 Latest stable release | Branch | Min CSI Version | Max CSI Version | Container Image | [Min K8s Version](kubernetes-compatibility.md#minimum-version) | [Max K8s Version](kubernetes-compatibility.md#maximum-version) | [Recommended K8s Version](kubernetes-compatibility.md#recommended-version) |
 --|--|--|--|--|--|--|--
+[external-attacher v3.0.0](https://github.com/kubernetes-csi/external-attacher/releases/tag/v3.0.0) | [release-3.0](https://github.com/kubernetes-csi/external-attacher/tree/release-3.0) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | k8s.gcr.io/sig-storage/csi-attacher:v3.0.0 | v1.17 | - | v1.17
```
Comment (msau42) [inline book/src/external-attacher.md:11]: @jsafrane csi-attacher release page says recommended 1.18 for v3.0.0 and v2.2.0, but it's 1.17 here. Which one is correct?

File: book/src/external-resizer.md [hunk 2]
```diff
@@ -8,7 +8,8 @@
 
 Latest stable release | Branch | Min CSI Version | Max CSI Version | Container Image | [Min K8s Version](kubernetes-compatibility.md#minimum-version) | [Max K8s Version](kubernetes-compatibility.md#maximum-version) | [Recommended K8s Version](kubernetes-compatibility.md#recommended-version) |
 --|--|--|--|--|--|--|--
-[external-resizer v0.5.0](https://github.com/kubernetes-csi/external-resizer/tree/v0.5.0)  | [release-0.5](https://github.com/kubernetes-csi/external-resizer/tree/release-0.5) |[v1.1.0](https://github.com/container-storage-interface/spec/releases/tag/v1.2.0) | - | quay.io/k8scsi/csi-resizer:v0.5.0 | v1.15 | - | v1.16
+[external-resizer v1.0.0](https://github.com/kubernetes-csi/external-resizer/tree/v1.0.0)  | [release-1.0](https://github.com/kubernetes-csi/external-resizer/tree/release-1.0) |[v1.2.0](https://github.com/container-storage-interface/spec/releases/tag/v1.2.0) | - | k8s.gcr.io/sig-storage/csi-resizer:v1.0.0 | v1.15 | - | v1.16
```
Comment (msau42) [inline book/src/external-resizer.md:11]: @gnufied csi-resizer release page says min v1.16. Is that correct?

File: book/src/snapshot-controller.md [hunk 3]
```diff
@@ -12,6 +12,7 @@ When Volume Snapshot is promoted to Beta in Kubernetes 1.17, the CSI external-sn
 
 Latest stable release | Branch | Min CSI Version | Max CSI Version | Container Image | [Min K8s Version](kubernetes-compatibility.md#minimum-version) | [Max K8s Version](kubernetes-compatibility.md#maximum-version) | [Recommended K8s Version](kubernetes-compatibility.md#recommended-version)
 --|--|--|--|--|--|--|--
+[external-snapshotter v3.0.0](https://github.com/kubernetes-csi/external-snapshotter/releases/tag/v3.0.0) | [release-3.0](https://github.com/kubernetes-csi/external-snapshotter/tree/release-3.0) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | k8s.gcr.io/sig-storage/snapshot-controller:v3.0.0 | v1.17 | - | v1.17
```
Comment (msau42) [inline book/src/snapshot-controller.md:15]: @xing-yang we'll want to add a new section/page for the validation webhook.

And maybe a top-level section for client libraries?

File: book/src/external-resizer.md (hunk 2 above)
Comment (gnufied) [reply book/src/external-resizer.md:11]: Yes I think 1.16 as minimum version makes sense. That is when CSI volume expansion went beta.

File: book/src/snapshot-controller.md (hunk 3 above)
Comment (xing-yang) [reply book/src/snapshot-controller.md:15]: Sure.  I can submit a PR for that

File: book/src/external-resizer.md (hunk 2 above)
Comment (msau42) [reply book/src/external-resizer.md]: updated

File: book/src/external-attacher.md (hunk 1 above)
Comment (jsafrane) [reply book/src/external-attacher.md:11]: I think it's 1.17, I remember fixing README in v3.0.0 to 1.17

**Pull request merged**